README.md

```diff
@@ -6,10 +6,12 @@
 
 `jextract` depends on the [C libclang API](https://clang.llvm.org/doxygen/group__CINDEX.html). To build the jextract sources, the easiest option is to download LLVM binaries for your platform, which can be found [here](https://releases.llvm.org/download.html) (a version >= 9 is required). Both the `jextract` tool and the bindings it generates depend heavily on the [Foreign Function & Memory API](https://openjdk.java.net/jeps/419), so a suitable [jdk 18 distribution](https://jdk.java.net/18/) is also required.
 
+The JDK used to build jextract can be specified with `-Pjdk=/path/to/jdk`. If that option is omitted, a build JDK will be downloaded automatically.
+
```

Comment:
Should we say JDK used to build jextract should be jdk 18 ?

Reply:
The jdk 18 requirement is listed in the paragraph before.